pkg/runtime/pubsub/bulksubscribe_events.go

```diff
@@ -50,8 +50,9 @@ func NewBulkSubscribeEnvelope(req *BulkSubscribeEnvelope) map[string]interface{}
 		reqID, err := uuid.NewRandom()
 		if err != nil {
 			bulkPSLogger.Warn("Unable to generate uuid for bulk subscribe request")
+		} else {
+			id = reqID.String()
```

Comment:
Necessary because if there's an error (kernel not having enough entropy), `reqID` can be nil